lib/node_modules/@stdlib/blas/ext/base/dcusumkbn/include/stdlib/blas/ext/base/dcusumkbn.h

```diff
@@ -34,7 +34,7 @@ extern "C" {
 /**
 * Computes the cumulative sum of double-precision floating-point strided array elements using an improved Kahan–Babuška algorithm.
 */
-void stdlib_strided_dcusumkbn( const int64_t N, const double sum, const double *X, const int64_t strideX, double *Y, const int64_t strideY );
+double stdlib_strided_dcusumkbn( const int64_t N, const double sum, const double *X, const int64_t strideX, double *Y, const int64_t strideY );
```

Comment:
This change in return type should be reverted.

```suggestion
void stdlib_strided_dcusumkbn( const int64_t N, const double sum, const double *X, const int64_t strideX, double *Y, const int64_t strideY );
```